### PR TITLE
Add release notes for #2393

### PR DESCRIPTION
**Pull request by IT-VBFK**
## IMPORTANT 

* [ ] If the PR touches the public API, the changes have been approved in a separate issue with the "api-approved" label.
* [ ] The code complies with the [Coding Guidelines for C#](https://www.csharpcodingguidelines.com/).
* [ ] The changes are covered by unit tests which follow the Arrange-Act-Assert syntax and the naming conventions such as is used [in these tests](../tree/develop/Tests/FluentAssertions.Equivalency.Specs/MemberMatchingSpecs.cs#L51-L430).
* [ ] If the PR adds a feature or fixes a bug, please update [the release notes](../tree/develop/docs/_pages/releases.md) with a functional description that explains what the change means to consumers of this library, which are published on the [website](https://fluentassertions.com/releases).
* [ ] If the PR changes the public API the changes needs to be included by running [AcceptApiChanges.ps1](../tree/develop/AcceptApiChanges.ps1) or [AcceptApiChanges.sh](../tree/develop/AcceptApiChanges.sh).
* [ ] If the PR affects [the documentation](../tree/develop/docs/_pages), please include your changes in this pull request so the documentation will appear on the [website](https://www.fluentassertions.com/introduction).
    * [ ] Please also run `./build.sh --target spellcheck` or `.\build.ps1 --target spellcheck` before pushing and check the good outcome

### CODE REVIEW

**Review by dennisdoomen — APPROVED** [review]
It looks a bit unconventional.

File: docs/_pages/releases.md [hunk 1]
```diff
@@ -22,7 +22,8 @@ sidebar:
 * Fixed that the `because` and `becauseArgs` were not passed down the equivalency tree - [#2318](https://github.com/fluentassertions/fluentassertions/pull/2318)
 * `BeEquivalentTo` can again compare a non-generic `IDictionary` with a generic one - [#2358](https://github.com/fluentassertions/fluentassertions/pull/23158)
 * Fixed that the `FormattingOptions` were not respected in inner `AssertionScope` - [#2328](https://github.com/fluentassertions/fluentassertions/pull/2328)
-* Capitalize `true` and `false` in failure messages - [#2390](https://github.com/fluentassertions/fluentassertions/pull/2390)
+* Capitalize `true` and `false` in failure messages - [#2390](https://github.com/fluentassertions/fluentassertions/pull/2390), but changed to pass them as argument to be able to be picked up by custom `BooleanFormatter` - [#2393](https://github.com/fluentassertions/fluentassertions/pull/2393)
```
Comment (ITaluone) [inline docs/_pages/releases.md:25]: May

```suggestion
* Capitalize `true` and `false` in failure messages - [#2390](https://github.com/fluentassertions/fluentassertions/pull/2390), but [#2393](https://github.com/fluentassertions/fluentassertions/pull/2393) changed to pass them as argument to be able to be picked up by custom `BooleanFormatter` - 
```

Maybe rephrase it a bit?

**Comment by IT-VBFK** [comment]
As I mentioned in 2393.. it's a propsal 😉

File: docs/_pages/releases.md (hunk 1 above)
Comment (jnyrup) [inline docs/_pages/releases.md:25]:
```suggestion
* Capitalize `true` and `false` in failure messages and make them formattable to a custom `BooleanFormatter` - [#2390](https://github.com/fluentassertions/fluentassertions/pull/2390), [#2393](https://github.com/fluentassertions/fluentassertions/pull/2393)
```

Comment (IT-VBFK) [reply docs/_pages/releases.md]: Yeah.. that's better 👍🏻

**Review by jnyrup — APPROVED** [review]
